nuxt/cypress/e2e/account.cy.ts

```diff
@@ -0,0 +1,17 @@
+describe('account test', () => {
+  beforeEach(() => {
+    cy.visit('/account')
+  })
+
+  // This is currently broken, due to nuxt3 redirects not working properly
+  // Should be fixed in nuxt v3.0.0-rc.12
+  // Expected behaviour: redirect to homepage, code 302
+  context('pages load correctly', () => {
+    it('check: redirect to log-in (when not logged in)', () => {
```

Comment:
```suggestion
    it('redirects to log-in when not logged in', () => {
```
The `it` is meant as start of a sentence and braces in the human / non coder readable string can be left out..